pkg/kubectl/cmd/proxy.go

```diff
@@ -76,7 +76,6 @@ The above lets you 'curl localhost:8001/custom/api/v1/pods'
 
 func RunProxy(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 	port := cmdutil.GetFlagInt(cmd, "port")
-	fmt.Fprintf(out, "Starting to serve on localhost:%d", port)
```

Comment:
This message is used in many docs to indicate that kubectl proxy is up.
We need to update those docs.